obspy/io/gcf/__init__.py

```diff
@@ -18,11 +19,83 @@
 The format will be determined automatically.
 
 >>> from obspy import read
->>> st = read("/path/to/20160603_1955n.gcf")
+>>> st = read("/path/to/20160603_1955n.gcf")  #doctest: +ELLIPSIS
 
-You can also specify the following keyword arguments that change the
-behavior of reading the file:
-* ``headonly=True``: Read only the header part, not the actual data
+GCF specific metadata is stored in ``stats.gcf``
+
+>>> for k, v in sorted(st[0].stats.gcf.items()):
+...     print("'%s': %s" % (k, str(v)))  #doctest: +NORMALIZE_WHITESPACE
+'FIC': -49378
+'RIC': -49312
+'blk': 0
+'digi': 0
+'gain': 1
+'stat': 0
+'streamID': 6018N4
+'sysType': 1
+'systemID': 6281
+'t_leap': False
+'ttl': 6
+
+Several keyword arguments are available to set parameters required by the returned 
+:class:`~obspy.core.stream.Stream` object as well as to control merging of data blocks
+on reading: ``networkcode``, ``stationcode``, ``locationcode``, ``bandcode``, 
+``instrumentcode``, ``channel_prefix``, ``blockmerge``, ``headonly``, 
+``cleanoverlap``, ``errorret``. They are passed to the :meth:`~obspy.io.gcf.core._read_gcf` 
+method so refer to this for details of each parameter.
+
+Function may raise TypeError on bad arguments (e.g. out of range) and IOError on failure
+to read/decode file or errors in data blocks in file (bad blocks may be returned by use
+of keyword argument ``errorret`` though, preventing IOError's raised by these)
+
+Writing
+-------
+Write data back to disc or a file like object using the
+:meth:`~obspy.core.stream.Stream.write` method of a
+:class:`~obspy.core.stream.Stream` or
+:class:`~obspy.core.trace.Trace` object.
+
+>>> st.write('GCF-filename.gcf', format='GCF') #doctest: +SKIP
+
+
+Several key word arguments are available to set required GCF specific header information and
+allowed missalignment of supported starttime of first data sample:
+``streamID``, ``systemID``, ``isLeap``, ``gain``, ``ttl``, ``digi``, ``sysType``, ``missalign``
```

Comment:
these kwargs will have to adhere to our naming scheme (snake case), so e.g. `streamID` will have to become `stream_id` or similar

Reply:
Sure, will fix it